src/main/java/com/ericsson/ei/handlers/RmqHandler.java

```diff
@@ -223,4 +225,22 @@ public void close() {
         }
     }
 
+    /**
+     * Return the server status of Rabbit MQ calculated on the number of active consumers.
+     *
+     * @return
+     */
+    public boolean isRabbitMQServerUp() {
+        boolean rabbitServerIsUp = false;
```

Comment:
assign this variable at declaration since there is no conditional reassignment done

Reply:
Done.